### ISSUE TITLE
Add V's Script to create FAKE CS DAILs as TIKL

### ISSUE THREAD

**Issue by C-Love** [Hennepin-County/MAXIS-scripts#403]
V created this to build the CSES Scrubber - we want to save it and maybe use it. 

Copied directly from a DHS repo working branch.

https://raw.githubusercontent.com/MN-Script-Team/DHS-MAXIS-Scripts/VKC-utility-to-create-CS-TIKLs/Script%20Files/UTILITIES/UTILITIES%20-%20CREATE%20FAKE%20CS%20DAILS%20AS%20TIKL.vbs

**Comment by C-Love** [comment]
I still need to test this. The code exists and I believe it should be fully functioning - completely Veronica's code.

**Comment by C-Love** [comment]
Updated to make sure this works most of the time. 

In testing I've figured out what is happening in this script. It will create a single TIKL with the date and amount already listed in a UNEA panel. It will not create TIKLs for more amounts or other dates. There is no dialog to allow for other entries or looping to pull more information from UNEA.

@IlseFerris do we want to consider making another issue for a later time to enhance this and make it so that we can create multiple TIKLs and maybe do it from a DIALOG instead of UNEA?

This script is working as it was intended but was created for a pretty limited scope. Planning to close this issue as is and only handle for enhancements in another issue/future work if we want.
